src/index.js

```diff
@@ -91,10 +95,22 @@ async function runtest(page, name) {
   // await runtest(page, "UploadPayrollStatus");
 
   await page.goto(
-    "https://srv136.services.gc.ca/ROE-RE/ROEWeb-REWeb/pro/Requests/Prints?org_id=" + orgId
+    "https://srv136.services.gc.ca/ROE-RE/ROEWeb-REWeb/pro/Requests/Prints?org_id=" +
+      orgId
   );
   await runtest(page, "RequestedPrintFiles");
 
+  await page.goto(
+    "https://srv136.services.gc.ca/ROE-RE/ROEWeb-REWeb/pro/Search/Issued?org_id=" +
+      orgId
+  );
+  await runtest(page, "SearchIssued");
+  await page.type("#SIN", "123456782");
+  await page.click(
```

Comment:
Use a `Promise.All`a for this so like this

```javascript
  await Promise.all([
    page.waitForNavigation(), // The promise resolves after navigation has finished
    page.click("button[type=submit]")
  ]);
```

*I just copy and pasted the above from an earlier line.

Otherwise you could have a race condition.
https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pagewaitfornavigationoptions

Reply:
I am running a `waitForNavigation` just not in a `Promise.all`